tests/clearpart_test.py

```diff
@@ -1,274 +1,240 @@
+""" Clearpart command related unit tests
+"""
 import unittest
 import mock
 
 import blivet
 import blivet.devices
 from blivet.devices.lib import Tags
-from pykickstart.constants import CLEARPART_TYPE_ALL, CLEARPART_TYPE_LINUX, CLEARPART_TYPE_NONE, CLEARPART_TYPE_LIST
-from parted import PARTITION_NORMAL
 from blivet.flags import flags
+from pykickstart.constants import CLEARPART_TYPE_ALL, CLEARPART_TYPE_LINUX
+from pykickstart.constants import CLEARPART_TYPE_NONE, CLEARPART_TYPE_LIST
+from parted import PARTITION_NORMAL
 
 DEVICE_CLASSES = [
     blivet.devices.DiskDevice,
     blivet.devices.PartitionDevice
 ]
 
+# access to protected attributes is needed for setting up the test environment
+# pragma pylint: disable=protected-access
 
-@unittest.skipUnless(not any(x.unavailable_type_dependencies() for x in DEVICE_CLASSES), "some unsupported device classes required for this test")
+
+@unittest.skipUnless(not any(x.unavailable_type_dependencies() for x in DEVICE_CLASSES),
+                     "some unsupported device classes required for this test")
 class ClearPartTestCase(unittest.TestCase):
+    """ Tests of clearpart command and its options
+    """
 
     def setUp(self):
         flags.testing = True
-        self.addCleanup(self._clean_up)
```

Comment:
Please keep the cleanup here, we need to be sure the flag change is reverted even if something in `setUp` fails.